openquakeplatform/bin/deploy.sh

```diff
@@ -624,6 +624,7 @@ oq_platform_install () {
     if [ "$GEM_IS_INSTALL" == "y" ]; then
         # Load our users. Default password must be changed
         openquakeplatform loaddata ${oqpdir}/common/fixtures/*.json
+        cat ${oqpdir}/common/fixtures/*.sql | openquakeplatform dbshell
```

Comment:
This also will break CI and any automated installation since it requires user interaction.